src/config/cloudtrail.json

```diff
@@ -0,0 +1,1006 @@
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBInstance', &log_content_parsed.responseElements.dBInstanceIdentifier, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBInstanceReadReplica', &dt_meid_rds_v2( \"aws.arn\"), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.arn",
+          "priority": 10,
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBInstanceReadReplica', &format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, log_content_parsed.responseElements.sourceDBInstanceIdentifier]), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBInstanceReadReplica', &log_content_parsed.responseElements.sourceDBInstanceIdentifier, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBSnapshot', &dt_meid_rds_v2( \"aws.arn\"), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.arn",
+          "priority": 10,
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBSnapshot', &format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, log_content_parsed.requestParameters.dBInstanceIdentifier]), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "if( log_content_parsed.eventName == 'CreateDBSnapshot', &log_content_parsed.requestParameters.dBInstanceIdentifier, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'CreateEventSubscription' && log_content_parsed.requestParameters.sourceType == 'db-cluster', &dt_meid_supporting_service_v2( 'Aurora', \"aws.arn\"), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.arn",
+          "priority": 10,
+          "pattern": "if( log_content_parsed.eventName == 'CreateEventSubscription' && log_content_parsed.requestParameters.sourceType == 'db-cluster', &format_arn('arn:{}:rds:{}:{}:cluster:{}', [partition, region, account_id, log_content_parsed.requestParameters.sourceIds[0] ]), &EmptyValue, @ )"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "if( log_content_parsed.eventName == 'CreateEventSubscription' && log_content_parsed.requestParameters.sourceType == 'db-cluster', &log_content_parsed.requestParameters.sourceIds[0], &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'CreateEventSubscription' && log_content_parsed.requestParameters.sourceType == 'db-instance', &dt_meid_rds_v2( \"aws.arn\"), &EmptyValue, @ )"
```

Comment:
This if function does "visit" on branch expression. So we have those ampersand-deferred expressions, right? Can we put non-deferred expression, like a normal string without ampersand or just "null"?

Reply:
Basically, can we have already evaluated "EmptyValue", so maybe put just "null" (without ampersand) so we don't have to evaluate &"EmptyValue" later?

Reply:
`Can we put non-deferred expression` no, it currently requires deferred, I'm not sure I want to fight to change that

 I had some trouble with plain null, that's why I ended with EmptyValue. Might recheck again if I will remember.

Reply:
No, if it does require deferred then I also see no point, let's leave this